wiki/Article_Styling_Criteria/en.md

```diff
@@ -28,15 +42,18 @@ Listed below are the properly-supported locales for the osu! wiki.
 | `pt-br.md` | Brazilian Portuguese | Português (Brasil) |
 | `ru.md` | Russian | Русский |
 | `sv.md` | Swedish | Svenska |
+| `th.md` | Thai | ไทย |
+| `tr.md` | Turkish | Türkçe |
+| `vi.md` | Vietnamese | Tiếng Việt |
 | `zh.md` | Chinese (Simplified) | 简体中文 |
 | `zh-tw.md` | Traditional Chinese (Taiwan) | 繁體中文（台灣） |
 
 Keep in mind, it is intended that the osu! wiki will try to give readers their selected language's version of an article (and fallback on using the English version if it is not available). If your language is not listed, you have two choices:
 
 - Append `?locale={langcode}` to the URL, where `{langcode}` is your language's two letter code in lowercase letters (this is the temporary solution). For example: `https://osu.ppy.sh/help/wiki/Welcome?locale=zh` will give you the Chinese version of the Welcome article.
-- Help translate the [`osu-web` repository](https://github.com/ppy/osu-web/) first (this is the permanent solution). If you need help translating the osu!web, please ask for help in the [osu!dev discord](https://discord.gg/ppy) in the `#osu-web` channel.
+- Help translate the [osu-web via Crowdin](https://crowdin.com/project/osu-web) (this is the permanent solution). If you need help translating the website, please ask for help in the [osu!dev discord](https://discord.gg/ppy) in the `#osu-web` channel.
```

Comment:
`discord` -> `Discord`
Idk, but shouldn't proper nouns be capitalized? If so, can you note it somewhere in a section called `Proper nouns` on `Capitalisation`?

Reply:
Hmm, I guess a `Proper nouns` section can be added. Though, Dean Herbert is probably the only one that will be used throughout the entire wiki.

Reply:
I assume you're specifically talking about trademarks, which can basically have any letter casing or unicode characters they want. Wikipedia has some rules about them, but to just keep it simple, I'll just list off the common ones... probably the social ones?

Reply:
Sure, the ones that osu! use. (i.e. Discord, Twitter, Github, Facebook, etc.)